index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <link rel="stylesheet" href="styles.css">
+        <title>first mobile app</title>
+    </head>
+
+    <body>
+        <section class="main-page">
+            <header class="main-header">
+                <nav>
+                    <div class="header-text">John Doe</div>
+                </nav>
+                <nav class="union">
+                    <div class="vector"></div>
+                    <div class="vector"></div>
+                    <div class="vector"></div>
+                </nav>
+            </header>
+            
+        <div class="page1">
```

Comment:
- [ ] The whole of this should be in a section tag and not inside a div. The section tag represents a generic standalone section of the document. So I would recommend you use it.

Reply:
done